pootle/apps/pootle_store/models.py

```diff
@@ -2138,9 +2138,14 @@ def _get_wordcount_stats(self):
             'translated': 0,
             'fuzzy': 0
         }
+
+        # only count files with an ext matching the project ext
+        proj_ext = self.translation_project.project.localfiletype
+
         # XXX: `order_by()` here is important as it removes the default
         # ordering for units. See #3897 for reference.
-        res = self.units.order_by().values('state') \
+        res = self.units.filter(store__file__endswith=proj_ext) \
```

Comment:
Most likely using `store__name__endswith=proj_ext` might be slightly faster since name is shorter than file, e.g. `tutorial/templates/tutorial.pot` vs just `tutorial.pot`.